docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -55,3 +51,37 @@ Captures screenshots for every step in the journey.
 
 *`-h, --help`*::
 Shows help for the `npx @elastic/synthetics` command.
+
+[discrete]
+[[elastic-synthetics-init-command]]
+== `elastic-synthetics init`
+
+Scaffold a new project using Elastic Synthetics.
+
+[source,sh]
+----
+npx @elastic/synthetics init <name-of-project>
+----
+
+
+[discrete]
+[[elastic-synthetics-push-command]]
+== `elastic-synthetics push`
+
+Bundle all your local journey files and create monitors in {kib}.
```

Comment:
Ok! This language was pulled from https://github.com/elastic/synthetics/releases/tag/v1.0.0-beta.26 so we should consider updating it there, too, if it could cause confusion.

Reply:
Agreed! I think it's worth updating it there, but I'd assume most people would look at these docs rather than the release notes given they're linked to a particular release, which ends up getting more and more distant in time.